images/jenkins-helmfile-version-query/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM alpine:3.11.5
+FROM alpine:3
```

Comment:
is this to fuzzy match everything after 3.xx.xx?

Reply:
yep